torch_patches/.torch_pin

```diff
@@ -0,0 +1 @@
+#36073
```

Comment:
This is your PR right?

Reply:
Yea it'll only pass with that PR. (and should be landed after that one).